.pylintrc

```diff
@@ -3,7 +3,8 @@ ignore=migrations
 load-plugins=pylint.extensions.no_self_use
 
 [MESSAGES CONTROL]
-disable=E1101,E1135,E1136,R0903,R0901,R0902,W0707,W0511,W0406,R0401,R0801,C3001,import-error
+# XXX remove `bad-classmethod-argument`
+disable=E1101,E1135,E1136,R0903,R0901,R0902,W0707,W0511,W0406,R0401,R0801,C3001,import-error,bad-classmethod-argument
```

Comment:
This here is the main reason the PR is marked as draft.

As `setUpTestData()` is a class method, pylint (rightfully) complains that the argument to the method should be `cls`, and not `self`. This involves substantially many more changes (for little benefit?).

I tried to find a way to disable this warning only in the bookwyrm/tests subdirectory, but it doesn't seem to be possible. I offer, for the time being, adding override directives to each test file, rather than leaving the global disable here.

Reply:
I think overrides in the file is fine, that's what I had been doing with invalid name. The CI I believe is not checking the test files, so it won't block PRs. This is a great change and having quicker test will be a big win :)

Reply:
Thank you for taking a look. I applied your suggestion!